src/PublicApiGenerator/AttributeFilter.cs

```diff
@@ -0,0 +1,27 @@
+using Mono.Cecil;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PublicApiGenerator
+{
+    internal sealed partial class AttributeFilter
+    {
+        private readonly HashSet<string> _excludedAttributes;
```

Comment:
Maybe use `SkipAttributeNames` directly without one more private field?

Reply:
This is not holding the same set of names that SkipAttributeNames holds, though.

Reply:
I mean to make SkipAttributeNames instance field and append all excluded attributes to it. SkipAttributeNames and excludedAttributes have the same meaning.

Reply:
Isn't that already what I'm doing? Where would the source of the append be stored?

Reply:
I can rename them `_skippedAttributes` and `DefaultSkippedAttributes` but that sounds like `DefaultSkippedAttributes` is not used if any attributes are specified to exclude.

Or `_excludedAttributes` and `AlwaysExcludedAttributes`?

Reply:
```c# 
internal sealed partial class AttributeFilter
{
     private readonly HashSet<string> SkippedAttributeNames = ...;
     public AttributeFilter(IEnumerable<string> excludedAttributes)
     {
          if (excludedAttributes != null)
                 SkippedAttributeNames  = new HashSet<string>(SkippedAttributeNames.Concat(excludedAttributes));
     }
}

Reply:
Just use one private instance field instead one static (existed) and one instance (new).

Reply:
I don't like this for two reasons:

- The SkippedAttributeNames.cs file is less parallel with the RequiredAttributeNames.cs file
- An unnecessary HashSet is built any time `excludedAttributes` is not null.

Why is having one fewer static fields worth that?

Reply:
Nothing special. The current version suits me.